### PR TITLE
Longest Palindromic Sequence python solution

### PR DESCRIPTION
**Pull request by Perruci**
Greetings,

A dynamic programming python approach on solving the Longest Palindromic Sequence has been implemented. Testing strings were added  into the main function.

The solution was inspired by online resources, as recommended by @Vitorvgc and targets the issue  #1319.

### CODE REVIEW

File: code/dynamic_programming/longest_palindromic_sequence/longest_palindromic_sequence.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+
+# Part of Cosmos by OpenGenus Foundation
+
+def longest_palindrome(text):
+    """ Find the maximum length of a palindrome subsequence
+
+    Dynamic Programming approach on solvind the longest palindromic sequence.
```
Comment (Vitorvgc) [inline code/dynamic_programming/longest_palindromic_sequence/longest_palindromic_sequence.py]: Just a little typo here. Besides that, looks good for me :+1:

Comment (Vitorvgc) [inline code/dynamic_programming/longest_palindromic_sequence/longest_palindromic_sequence.py]: Just a little typo here. Besides that, looks good for me :+1:

Comment (Vitorvgc) [inline code/dynamic_programming/longest_palindromic_sequence/longest_palindromic_sequence.py]: Just a little typo here. Besides that, looks good for me :+1:

**Comment by AdiChat** [comment]
Looks great 😍  

Hope you are enjoying your journey with **OpenGenus**

**Pull request merged**